src/BigInteger.php

```diff
@@ -386,6 +386,28 @@ public function gcd($that) : BigInteger
         return new BigInteger($value);
     }
 
+    /**
+     * Returns the square root number of this number and the given one.
+     *
+     * @param BigNumber|number|string $that The number to add. Must be convertible to a BigInteger.
+     *
+     * @return BigInteger The result.
+     *
+     * @throws MathException If the number is not valid, or is not convertible to a BigInteger.
+     */
+    public function sqrt($that) : BigInteger
```

Comment:
This method should have no parameter. We're returning the square root of this number, not another one.